REQUIREMENTS.md

```diff
@@ -0,0 +1,128 @@
+# Article Requirements
+
+These are the stylistic and informational requirements for articles
+
+## Table of contents
+
+-   [Last Update Date](#last-update-date)
+    -   [Articles With No Sections](#articles-with-no-sections)
+    -   [Articles With Sections](#articles-with-sections)
+-   [Citations](#citations)
+    -   [Whole Article](#whole-article)
+    -   [Sections](#sections)
+
+## Last Update Date
+
+As most of this information is time sensitive, listing the last updated date is important.
+
+Dates need to be in the format `yyyy/mm/dd` for example, `2022/12/25`
```

Comment:
How we will maintain the last updated info to be **updated**? I would recommend to add in the list of check marks when someone send a PR. That way, both committer and reviewer will be responsible for the details. 

**If you updated an Article, did you update the last updated section?**

Reply:
I would think that is implied by the "did you read the article requirements" question

Reply:
adding another check mark will make sure, it has been done. It was just something I thought.

Reply:
yeah it is implied by the `did you read the article requirements` question. There may end up being a bunch of article requirements with their own rules and conventions. If we put them all as check boxes in the PR template, it can get complicated. 

I think having a separate document is the way to go.

Reply:
to be honest, I don't think three or four check boxes will hurt.

- [ ] Read the Article Requirements 
- [ ] prettier 
- [ ] last updated

Reply:
Yeah but `last updated` is included in `Read the Article Requirements`. I just think its redundant. Not to mention we can change it if it stops working out for us. I do like the `prettier` idea though

Reply:
The "prettier" box is solid IMO, but I still think Last Updated is implied by Article Requirements. Its not an issue with the number of boxes we would have now, its that if we start listing the article requirements in the PR template, then eventually we'll have dozens of them.

Reply:
agree!!